### PR TITLE
Remove void break from fluid tanks and chests

### PR DESCRIPTION
**Pull request by NexusNull**
Previously, sneaking while breaking a fluid tank would void the contents of the tank, leading to potential loss of valuable resources. However, this behavior is not consistent with other blocks, such as chests and furnaces, which retain their contents when broken while sneaking.

This commit removes the sneak breaking voiding behavior for fluid tanks, bringing them in line with other blocks and reducing the risk of unintended loss for players.

### CODE REVIEW

**Review by greesyB — COMMENTED** [review]
I am against this change. Ability to void contents on harvest is useful and should be kept.

File: src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalChestBase.java [hunk 1]
```diff
@@ -361,7 +359,6 @@ public boolean onRightclick(IGregTechTileEntity aBaseMetaTileEntity, EntityPlaye
 
     @Override
     public void onLeftclick(IGregTechTileEntity aBaseMetaTileEntity, EntityPlayer aPlayer) {
```
Comment (miozune) [inline src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalChestBase.java:361]: just remove this override

File: src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalTankBase.java [hunk 2]
```diff
@@ -252,7 +250,6 @@ public boolean onRightclick(IGregTechTileEntity aBaseMetaTileEntity, EntityPlaye
 
     @Override
     public void onLeftclick(IGregTechTileEntity aBaseMetaTileEntity, EntityPlayer aPlayer) {
```
Comment (miozune) [inline src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalTankBase.java:252]: same here

File: src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalChestBase.java [hunk 3]
```diff
@@ -50,8 +49,7 @@ public GT_MetaTileEntity_DigitalChestBase(int aID, String aName, String aNameReg
             "This Chest stores " + GT_Utility.formatNumbers(commonSizeCompute(aTier)) + " Blocks",
             "Use a screwdriver to enable",
             "voiding items on overflow",
-            "Can keep its contents when harvested",
-            "Sneak when harvesting to void its contents"
+            "Will keep its contents when harvested",
```
Comment (miozune) [inline src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalChestBase.java:52]: Also edit `GT5U.machines.digitaltank.tooltip2` on `en_US.lang`.

**Review by miozune — COMMENTED** [review]
I'm in favor of this change, we have "void all" button on GUI now.

**Comment by OneEyeMaker** [comment]
I agree with @greesyB that ability to void container can be very useful. But I also can understand intent of author of PR - it's not intuitive that sneaking leads to voiding resources. And sometimes you press shift button accidentally.

I suggest adding UI button, the task of which will be to void contents of container. This UI feature should also re-ask: "Are you really sure to void contents of container?" (to prevent accidental misclicks).

In my opinion, current PR can be merged only in presence of this (or similar) feature.

**Comment by miozune** [comment]
> I suggest adding UI button, the task of which will be to void contents of container.

We already have it


> This UI feature should also re-ask: "Are you really sure to void contents of container?" (to prevent accidental misclicks).

I think it's redundant. The button is placed away from other three.

**Comment by greesyB** [comment]
I forgot about this button since it is relatively new. In that case while I still like sneak to void on harvest for convenience without needing to open GUI I am indifferent if others do not.

**Comment by POPlol333** [comment]
I'm for it  since sneaking to break can often happen when you fly or in some other situation leading to unwanted void, and yeah if you want to void there the button

File: src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalChestBase.java (hunk 3 above)
Comment (NexusNull) [reply src/main/java/gregtech/common/tileentities/storage/GT_MetaTileEntity_DigitalChestBase.java:52]: tooltip2 has been removed

**Comment by NexusNull** [comment]
The PR is ready for review again.

**Review by Connor-Colenso — APPROVED** [review]

**Pull request merged**